packages/mobx/__tests__/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "../tsconfig.json",
```

Comment:
@Nokel81 Could you describe why a new config file was introduced here?

Reply:
This is required because otherwise `strict` mode was not being applied to the typescript test files which causes `string | undefined` to be flattened to `string` thus making the tests not pass.